source/channels/channels-settings.rst

```diff
@@ -206,6 +206,14 @@ Select which language Mattermost displays in the user interface. Options include
 - 中文 (繁體) - Traditional Chinese
 - 日本語 - Japanese
 
+Advanced settings
+~~~~~~~~~~~~~~~~~
+
+Using the Mattermost mobile app, you can delete local Mattermost files from your device. Access **Settings** by tapping on your profile picture.
```

Comment:
Not sure if clear all the different clicks that have to be made. You have to click the profile picture and then click on Settings. But maybe it is clear as it is phrased right now.

Reply:
This is a standard approached we're using to walk mobile users to their end goals. We'll likely revisit this approach in a future release.